fullcontact/api/base/enrich_base_api.py

```diff
@@ -28,7 +30,7 @@ def _enrich_endpoint(self) -> str:
 
     @property
     @abstractmethod
-    def _enrich_request(self):
+    def _enrich_request(self) -> BaseSchema:
```

Comment:
Is it worth calling this `BaseSchemaRequest`? I don't feel super strongly about that, just bringing that up.

Reply:
Maybe `BaseRequestSchema` would suit it better, as this the base for all API request schemas.